### PR TITLE
Demo users module

### PR DESCRIPTION
**Pull request by aschmoe**

### CODE REVIEW

**Comment by aschmoe** [comment]
There's an issue where this hook_install is being run before the config import that adds user fields... there should be a nicer way to control the order of operations

File: config/sync/core.extension.yml [hunk 1]
```diff
@@ -20,6 +20,7 @@ module:
   filter: 0
   flash_blocks: 0
   flash_login: 0
+  flash_demo_users: 0
```
Comment (nerdstein) [inline config/sync/core.extension.yml:23]: This is where you set the module weight. This would affect it's order in the runtime

**Comment by nerdstein** [comment]
We need to remove the demo content in the install script as well. Just keep it in one place

**Comment by nerdstein** [comment]
Also, update the content block on the login page

**Comment by jlyon** [comment]
I think the demo content is used (and then deleted) in the Selenium tests, so we should make sure that we update those.

**Pull request merged**